src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public int getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        System.out.println(color.getRandomColor());
+        Random random = new Random();
```

Comment:
You should avoid creating a new instance of Random every time you call the method. Instead, make it a private static final field of the class to ensure only one instance is used throughout.